### PR TITLE
Update spin formula to 1.4.2

### PR DESCRIPTION
**Pull request by ThorstenHans**
This PR bumps the `spin` formula to `1.4.2`

### CODE REVIEW

File: Formula/spin.rb [hunk 1]
```diff
@@ -1,26 +1,26 @@
 class Spin < Formula
   desc "Open-source tool for building and running serverless WebAssembly applications"
   homepage "https://developer.fermyon.com/spin"
-  version "1.4.1"
+  version "1.4.3"
```
Comment (vdice) [inline Formula/spin.rb]:
```suggestion
  version "1.4.2"
```

**Review by vdice — COMMENTED** [review]
Thanks @ThorstenHans!  Spotted a typo in the version string, otherwise looks great to me.

File: Formula/spin.rb (hunk 1 above)
Comment (ThorstenHans) [reply Formula/spin.rb]: Oha! Thanks @vdice for finding this critical typo!!! My bad, I updated the PR correspondingly. 🙏🏻

**Review by vdice — APPROVED** [review]
LGTM from me!  Pinging @kate-goldenring to double-check all is in order.

**Review by kate-goldenring — APPROVED** [review]

**Pull request merged**